foxglove/cmd/utils.go

```diff
@@ -150,11 +150,37 @@ func AddFormatFlag(cmd *cobra.Command, format *string) {
 		format,
 		"format",
 		"",
-		"table",
+		"",
+		"render output in specified format (table, json, csv)",
+	)
+}
+
+func AddJsonFlag(cmd *cobra.Command, isJsonFormat *bool) {
+	cmd.PersistentFlags().BoolVar(
+		isJsonFormat,
+		"json",
+		false,
 		"render output in specified format (table, json, csv)",
 	)
 }
 
+// Ensure --json alias is not conflicting with --format's value
+func EnsureFormatFlagsNotConflicting(formatFlagValue string, jsonFlagValue bool) string {
```

Comment:
Maybe call this `ResolveFormat()`, or something that indicates the return value. Or name the return value.